p2p/discovery/option.go

```diff
@@ -33,6 +33,7 @@ func (opt DHTConfig) getLibp2pRawOptions() ([]libp2p_dht.Option, error) {
 		opts = append(opts, dsOption)
 	}
 
+	opts = append(opts, libp2p_dht.Concurrency(1))
```

Comment:
Why set concurrency to 1?

Reply:
Because the size of this parameter directly affects the frequency of calls to queryPeer in Discovery's functional features, and queryPeer is the main person responsible for taking up memory after I read kad-dht implementation code. The default value of this parameter is 10, and changing it to 1 is a relatively safe value for the 4G memory size of the Validator node, which I think is 1. Of course, I can also try to increase this value and test the memory performance again.

Reply:
what's the downside of setting it to 1 v.s. larger number like 10?

Reply:
According to the implementation code of libp2p-kad-dht, it is to affect the number of concurrency of queries on the same path, it reduces the performance of queries on the same path, on the same path, it will go to the number of concurrency of N = 1 Peer to find information. In a nutshell: it affects the performance of Discovery, but it is better than directly Disable Discovery.

Reply:
ok, to test the performance impact of setting it to 1 vs 10. If there is no problem with 1, we can go with 1 then.